<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarkEditWidget</class>
 <widget class="QWidget" name="BookmarkEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QCheckBox" name="AutoJoin_">
     <property name="text">
      <string>Autojoin</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="HumanReadable_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>Name_</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Name_"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nicknameLabel_">
       <property name="text">
        <string>Nickname:</string>
       </property>
       <property name="buddy">
        <cstring>Nickname_</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Nickname_"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="serverLabel">
       <property name="text">
        <string>Server:</string>
       </property>
       <property name="buddy">
        <cstring>Server_</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Server_"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port:</string>
       </property>
       <property name="buddy">
        <cstring>Port_</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="Port_">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="encodingLabel">
       <property name="text">
        <string>Encoding:</string>
       </property>
       <property name="buddy">
        <cstring>Encoding_</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="Encoding_"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="channelLabel">
       <property name="text">
        <string>Channel:</string>
       </property>
       <property name="buddy">
        <cstring>Channel_</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="Channel_"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Channel password:</string>
       </property>
       <property name="buddy">
        <cstring>Password_</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="Password_">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="useSSLLabel">
       <property name="text">
        <string>Use SSL:</string>
       </property>
       <property name="buddy">
        <cstring>SSL_</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="SSL_"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="ServerPassword_">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Server password:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
